### ISSUE TITLE
After upgrading to tailscale 1.8.7, all name servers removed from /etc/resolv.conf 

### ISSUE THREAD

**Issue by britannic** [tailscale/tailscale#2170]
After upgrading to tailscale 1.8.7, all name servers were removed from /etc/resolv.conf.

```
tailscale version
1.8.7
  tailscale commit: e1571ed7e0e84aed16f267be24fe408150109af6
  other commit: 2f595b1b68943e7c7670bb9c10147d81d034f6fd
  go version: go1.16.4-ts1b677aa
```

MagicDNS was not being used. Tailscale configuration has been working fine for months.

I tried adding name servers in the admin console, but it didn't update /etc/resolv.conf on our servers. Our mail server was hardest hit, as it lost DNS resolution.

Here is the resolv.conf file before upgrading:
```
# This file is managed by man:systemd-resolved(8). Do not edit.
#
# This is a dynamic resolv.conf file for connecting local clients to the
# internal DNS stub resolver of systemd-resolved. This file lists all
# configured search domains.
#
# Run "resolvectl status" to see details about the uplink DNS servers
# currently in use.
#
# Third party programs must not access this file directly, but only through the
# symlink at /etc/resolv.conf. To manage man:resolv.conf(5) in a different way,
# replace this symlink by a static file or a different symlink.
#
# See man:systemd-resolved.service(8) for details about the supported modes of
# operation for /etc/resolv.conf.

nameserver 127.0.0.53
options edns0 trust-ad
search hrss.local
```

This is the resolv.conf file after the upgrade:
```
# resolv.conf(5) file generated by tailscale
# DO NOT EDIT THIS FILE BY HAND -- CHANGES WILL BE OVERWRITTEN

search hrss.local
```

**Comment by DentonGentry** [comment]
I believe this is https://github.com/tailscale/tailscale/issues/2127, a race condition between Tailscale and systemd-resolved at boot. It will be fixed in the 1.10 release expected next week. If you're willing to install a build from the unstable track https://pkgs.tailscale.com/unstable/, 1.9.185 contains the fix now.

**Comment by britannic** [comment]
Thanks for the update @DentonGentry. Our workaround was to use:
```
tailscale up --accept-dns=false
```
